### PR TITLE
Fix wrong tr in aapl

### PR DESCRIPTION
**Pull request by phkroll**

### CODE REVIEW

File: bin/mozilla/aa.pl [hunk 1]
```diff
@@ -1896,6 +1896,7 @@ sub search {
 	      $vc
 	      $invnumber
           $currency
+          <tr>
```
Comment (phkroll) [inline bin/mozilla/aa.pl:1899]: @ledger123 I'm not quite sure if it is the right place, or if it should be before $vc. But I guess it should be here, didn't know where it is displayed.

**Comment by armaghansaqib** [comment]
Hi Philipp, please commit without fixing this tr and let me know and then I shall see, fix it, check it in browser and commit it.

This is on AR/AP transactions search screen and even with incorrect tr it works ok as most browsers forgive such small things.

**Comment by phkroll** [comment]
By without you are talking about the one I wasn't 100% sure about?

I checked now the search place and I'm now sure that it's the correct place, since it is the opening tr for the transdatefrom/transdateto fields. While the upper $vc, $invnumber, $currency are variables which hold some HTML I didn't realize before.

File: bin/mozilla/aa.pl (hunk 1 above)
Comment (ledger123) [reply bin/mozilla/aa.pl:1899]: This was added by you. Right. We need to remove this from here and then change <tr> to </tr> on line 1811 of develop as of now. I am doing both things.

**Pull request closed without merging**